code/core/src/core-server/build-dev.ts

```diff
@@ -190,13 +190,13 @@ export async function buildDevStandalone(
     ...options,
   });
 
-  const features = await presets.apply('features');
-  global.FEATURES = features;
+  const flags = await presets.apply('flags');
+  global.FLAGS = flags;
 
   const fullOptions: Options = {
     ...options,
     presets,
-    features,
+    flags,
```

Comment:
we still want to pass features, don't we? You're passing both flags in `build-static` below at least.